sklearn/externals/_lobpcg.py

```diff
@@ -22,11 +22,12 @@
 
 import warnings
 import numpy as np
-from scipy.linalg import (inv, eigh, cho_factor, cho_solve,
-                          cholesky, LinAlgError)
+from scipy.linalg import inv, eigh, cho_factor, cho_solve, cholesky, LinAlgError
 from scipy.sparse.linalg import aslinearoperator
 from numpy import block as bmat
 
+from ..utils.fixes import _eigh
```

Comment:
Apparently this is introducing a circular dependency in the doc builds...

Reply:
Indeed, for the oldest version of SciPy, we are doing circular imports. Let me try to solve this one (I will revert `black` as well).